include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -227,6 +227,67 @@ def TTG_InsertSliceOp : TTG_Op<"insert_slice",
 }
 
 
+def TTG_ViewSliceOp : TTG_Op<"view_slice",
+                                [AttrSizedOperandSegments,
+                                //  ResultsAreSharedEncoding,
```

Comment:
I'm thinking if we want to add a trait to check that the slice and the parent tensor can have the same layout.